package.json

```
@@ -2864,6 +2864,118 @@
 												]
 											}
 										]
+									},
+									{
+										"key": "languageId:ruby",
+										"name": "Ruby",
+										"type": "bindings",
+										"bindings": [
+											{
+												"key": "g",
+												"name": "+Go to",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "d",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "e",
+														"name": "Go to errors/problems",
+														"type": "command",
+														"command": "workbench.action.problems.focus"
+													},
+													{
+														"key": "g",
+														"name": "Go to definition",
+														"type": "command",
+														"command": "editor.action.revealDefinition"
+													},
+													{
+														"key": "i",
+														"name": "Find symbol in file",
+														"type": "command",
+														"command": "workbench.action.gotoSymbol"
+													},
+													{
+														"key": "r",
+														"name": "Peek references",
+														"type": "command",
+														"command": "editor.action.referenceSearch.trigger"
+													},
+													{
+														"key": "D",
+														"name": "Peek definition",
+														"type": "command",
+														"command": "editor.action.peekDefinition"
+													},
+													{
+														"key": "I",
+														"name": "Find symbol in project",
+														"type": "command",
+														"command": "workbench.action.showAllSymbols"
+													},
+													{
+														"key": "R",
+														"name": "Find all references",
+														"type": "command",
+														"command": "references-view.findReferences"
+													}
+												]
+											},
+											{
+												"key": "=",
+												"name": "+Format",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format document",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "d",
+														"name": "+Format document with",
```

```suggestion
														"name": "Format document",
```
It seems like this is from the go major mode, I will fix the wording there as well

---

You got me, I was a cheeky copy-paster ;) Updated

---

> I have seen this in a few places [[1](https://github.com/VSpaceCode/VSpaceCode/pull/159/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R2938-R2944)][[2](https://github.com/VSpaceCode/VSpaceCode/blob/v0.8.5/package.json#L2126-L2131)] and I think I figure it out why it is name `d` it's probably means document.
> 
> That makes me rethink about the whole `=` format in our bindings. I am thinking we should have unified view on the format commands, and opened an issue https://github.com/VSpaceCode/VSpaceCode/issues/160. Since we have multiple major modes PR in flights. I suggest we merge them and fix it as part of https://github.com/VSpaceCode/VSpaceCode/issues/160.

Just FYI, since I commented on the similar issue with another PR